test/layout/test_units.py

```diff
@@ -0,0 +1,57 @@
+from pathlib import Path
+
+import pytest
+
+from fpdf import FPDF
+from test.conftest import assert_pdf_equal
+
+HERE = Path(__file__).resolve().parent
+
+
+def test_unit_default(tmp_path):
+    """Test creating a PDF with no unit"""
+    pdf = FPDF()
+    pdf.set_font("Helvetica")
+    pdf.add_page()
+    pdf.cell(w=10, h=10, txt="default")
+    assert_pdf_equal(pdf, HERE / "unit_default.pdf", tmp_path)
+
+
+def test_unit_adjustment(tmp_path):
```

Comment:
Just a detail: could you please have the function name & reference PDF file name match?

Reply:
Done